guides/common/modules/con_projectserver-operating-system.adoc

```diff
@@ -0,0 +1,7 @@
+[id="ProjectServer-Operating-System_{context}"]
+= {ProjectServer} Operating System
+
+{Project} has packages for {EL} 8, Debian 11 and Ubuntu 20.04.
+Katello plug-in packages, which provide content management capabilities, are only available for {EL}.
+
+The only architecture {Project} has packages for is x86_64.
```

Comment:
```suggestion
{Project} is only packaged for the x86_64 archiecture.
```

Or maybe "{Team} only packages {Project} for ..."?

Reply:
+1 for "{Team} only packages"